Added a beta message to the Python AI integrations (#9896)
getsentry/sentry-docs: docs/platforms/python/integrations/openai/index.mdx

description: "Learn about using Sentry for OpenAI."
---

<Alert level="info" title="Beta">

> The support for **OpenAI** is in its beta phase. It is not yet fully ready for the public.
> 
> We are working on supporting different AI libraries (see [GitHub discussion](https://github.com/getsentry/sentry-python/discussions/3007))
> 
> If you want to try the beta features and are willing to give feedback, please shoot us a line on [Discord](https://discord.com/invite/Ww9hbqr).

Reviewer: ```suggestion
The support for **OpenAI** is in its beta phase.

We are working on supporting different AI libraries (see [GitHub discussion](https://github.com/getsentry/sentry-python/discussions/3007)).

If you want to try the beta features and are willing to give feedback, please let us know on [Discord](https://discord.com/invite/Ww9hbqr).
```
